app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  has_many :likes, dependent: :destroy
+  has_many :posts, dependent: :destroy
+  has_many :comments, dependent: :destroy
```

Comment:
- [ ] [ Required ] I think it would be better if you specified the exact field that references the user id in the posts, comments, and likes tables. Since the posts, likes, and comments tables use author_id to refer to the user id you have to specify this relation also in the user model. Please see the code sample below for more details on this and rectify it.
```
  has_many :posts, foreign_key: 'author_id'
  has_many :comments, foreign_key: 'author_id'
  has_many :likes, foreign_key: 'author_id'
```